contracts/neutron-interchain-txs/src/error.rs

```diff
@@ -0,0 +1,48 @@
+use cosmwasm_std::StdError;
+use protobuf::Error as ProtobufError;
+use thiserror::Error;
+
+pub type ContractResult<T> = Result<T, ContractError>;
+
+#[derive(Error, Debug, PartialEq)]
+pub enum ContractError {
```

Comment:
We have the same error [here](https://github.com/neutron-org/neutron-contracts/blob/b7ba2f9d9585a86f702284e9a4d06bd834eb257c/packages/interchain_queries/src/error.rs#L9). Let's define this error as common Neutron-SDK error (not `ContractError`, maybe `NeutronError` looks better)? And reuse it everywhere.

Reply:
And store it in some more common package (not interchain-txs and not interchain-queries)?

Reply:
I feel like we need an error package for this one. Or even separate task to generalize neutron errors?